common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -73,24 +77,25 @@ public static void info(String msg) {
         LOGGER.info(msg);
     }
 
-    public static void onResourcesFinishedLoading() {
-        if (featuresInited) return;
```

Comment:
Why remove this?

Reply:
Yeah, you can't do that. This is a guard to make sure we only do the init once. Resources can reload any number of times, we only want the first.

Also, the idea with initFeatures() was to create an abstraction barrier, to create for ourselves a way of doing "init when resources are available". I am not fond of you removing it. But maybe it needs comments to explain why it is as it is.